modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -0,0 +1,858 @@
+              command: !Sub
+                - >-
+                  watchmaker --log-level debug --log-dir /var/log/watchmaker --no-reboot --salt-states None
+                  ${local_UseWamConfig}
+                  ${local_UseEnvironment}
+                  ${local_UseOUPath}
+                  ${local_UseAdminGroups}
+                  ${local_UseAdminUsers}
+                -
+                  local_UseWamConfig: !If [UseWamConfig, !Sub '--config ${WatchmakerConfig}', '']
+                  local_UseEnvironment: !If [UseEnvironment, !Sub '--env ${WatchmakerEnvironment}', '']
+                  local_UseOUPath: !If [UseOuPath, !Sub '--ou-path ${WatchmakerOuPath}', '']
+                  local_UseAdminGroups: !If [UseAdminGroups, !Sub '--admin-groups ${WatchmakerAdminGroups}', '']
+                  local_UseAdminUsers: !If [UseAdminUsers, !Sub '--admin-users ${WatchmakerAdminUsers}', '']
+      ToggleCfnInitUpdate: !Ref ToggleCfnInitUpdate
+    Properties:
+      AssociatePublicIpAddress: !If 
+        - AssignPublicIp
+        - true
+        - false
+      BlockDeviceMappings:
+        - DeviceName: !Sub 
+            - '/dev/${local_Distro2RootDevice}'
+            - local_Distro2RootDevice: !FindInMap [Distro2RootDevice, !Ref AmiDistro, DeviceName]
+          Ebs:
+            DeleteOnTermination: true
+            VolumeType: gp2
+        - !If 
+          - CreateAppVolume
+          - DeviceName: /dev/xvdf
+            Ebs:
+              DeleteOnTermination: true
+              VolumeSize: !Ref AppVolumeSize
+              VolumeType: !Ref AppVolumeType
+          - !Ref 'AWS::NoValue'
+      IamInstanceProfile: !If 
+        - AssignInstanceRole
+        - !Ref InstanceRole
+        - !Ref 'AWS::NoValue'
+      ImageId: !Ref AmiId
+      InstanceType: !Ref InstanceType
+      KeyName: !Ref KeyPairName
+      SecurityGroups: !Ref SecurityGroupIds
+      UserData: 
+        !Base64
+          Fn::Sub:
+            - |
+              Content-Type: multipart/mixed; boundary="===============3585321300151562773=="
+              MIME-Version: 1.0
+              --===============3585321300151562773==
+              Content-Type: text/cloud-config; charset="us-ascii"
+              MIME-Version: 1.0
+              Content-Transfer-Encoding: 7bit
+              Content-Disposition: attachment; filename="cloud.cfg"
+
+              #cloud-config
+              ${local_CreateAppVolume}
+
+              --===============3585321300151562773==
+              Content-Type: text/x-shellscript; charset="us-ascii"
+              MIME-Version: 1.0
+              Content-Transfer-Encoding: 7bit
+              Content-Disposition: attachment; filename="script.sh"
+
+              #!/bin/bash -xe
+
+              # CFN LaunchConfi Update Toggle
+              ${ToggleNewInstances}
+
+              # Export AWS ENVs
+              test -r /etc/aws/models/endpoints.json && export
+              AWS_DATA_PATH=/etc/aws/models || true
```

Comment:
With the above suggestion, just delete this line.

Reply:
fixed